index.html

```diff
@@ -1064,7 +1064,7 @@ <h2>Radio</h2>
 					<ts-button onclick="closeBigModal()">Close!</ts-button>
 				</div>
 			</ts-modal>
-			<ts-modal id="medium-modal" data-title="Available Apps" data-size="medium">
+			<ts-modal id="medium-modal" hide-header data-size="medium">
```

Comment:
I am not sure it is a good idea or not. Can you use `data-title` to hide or show the header?

Reply:
Since the close button is also in the header (and also the possibility that we need more stuff in header ) I decided to have separate property for hiding the header. 👍